crucible-concurrency/DesignNotes.md

```diff
@@ -0,0 +1,185 @@
+# Crucibles/Cruces: concurrency-enabled symbolic simulation
```

Comment:
Is there a place that explains the meaning behind the name "Crucibles"? I was sort of expecting it to more closely resemble the name of the package, `crucible-concurrency`.

Reply:
I guess it's the plural of crucible (multiple threads, etc...). `crucible-concurrency` seems like a mouthful for text, but you're right it could at least be explained. Now that you point it out, the cabal file should probably be named crucible-concurrency, etc.